### PR TITLE
Feature/discordjs 13 migration

### PR DESCRIPTION
**Pull request by dcepil**
the repo needs a lot of cleanup work & existing services need refactoring - this is a very crude migration - I only tested 8ball and coinflip commands, expect broken things

warning - you will need to remove your db.sqlite3 file and run `npm i` to run migrations

requires at least node 16.6 as per v13 requirements

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
     "env:new": "node -p \"require('fs').copyFile('.env.copy', '.env', (err)=>{if(err){throw err}})\"",
     "test": "jest --forceExit --coverage --verbose",
     "test:legacy": "mocha test",
-    "start": "node dist/index.js",
-    "postinstall": "concurrently \"npm run db:migrate\" \"npm run build:src\" \"npm run env:new\" --max-processes 1",
+    "start": "concurrently \"npm run build:src\" \"node dist/index.js\" --max-processes 1",
+    "start:nobuild": "node dist/index.js",
+    "cloneEnv": "node dist/scripts/cloneEnv.js --clone",
```
Comment (DarkStoorM) [inline package.json]: We don't need `cloneEnv` anymore 🏃‍♂️

**Review by DarkStoorM — CHANGES_REQUESTED** [review]
👮‍♂️

File: .sequelizerc [hunk 2]
```diff
@@ -3,5 +3,5 @@ const path = require('path');
 module.exports = {
     "config": path.resolve("src", "config", "db-config.json"),
     "migrations-path": path.resolve("db", "migrations"),
-    "seders-path": path.resolve("db", "seeders")
+    "seeders-path": path.resolve("db", "seeders")
```
Comment (VuongRon) [inline .sequelizerc:6]: 👏 🆗 👍

**Review by VuongRon — APPROVED** [review]
🙆‍♂️

**Pull request merged**